pci/src/device.rs

```diff
@@ -46,6 +49,9 @@ pub trait PciDevice: BusDevice {
     /// The device may write to `irq_evt` to trigger an interrupt.
     fn assign_irq(&mut self, _irq_evt: Arc<IrqClosure>, _irq_num: u32, _irq_pin: PciInterruptPin) {}
```

Comment:
Once you added msix then I'd change this to assign_pin_irq for clarification. Also _irq_evt to _irq_cb to keep consistent.

Reply:
Sounds good!